scripts/docker_build.sh

```diff
@@ -23,10 +23,12 @@
 GIT_ROOT=$(git rev-parse --show-toplevel)
 GIT_VERSION=$(git describe --abbrev=10 --always --tags --dirty)
 ORT_DOCKER=${ORT_DOCKER:-ort}
+ARCH_TRANSLATION=""
 
 echo "Setting ORT_VERSION to $GIT_VERSION."
-DOCKER_BUILDKIT=1 docker build -f "$GIT_ROOT"/Dockerfile \
+docker buildx build -f "$GIT_ROOT"/Dockerfile \
```

Comment:
Switching to `buildx` is a rather big under-the-hood change that deserves its own commit with a thorough explanation.